.storybook/preview.js

```diff
@@ -1,20 +1,29 @@
 import 'core-js/stable' // to polyfill ECMAScript features
 import 'regenerator-runtime/runtime' // to use transpiled generator functions
+// *** TODO: stop using css-loader in package.json
 import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-loader
 import Vue from 'vue'
 import Vuetify from 'vuetify'
 import 'vuetify/dist/vuetify.min.css'
 import Vuelidate from 'vuelidate'
+import { addDecorator } from '@storybook/vue';
 
 import '@/assets/styles/base.scss'
 import '@/assets/styles/layout.scss'
 import '@/assets/styles/overrides.scss'
 
-Vue.use(Vuetify, {
-  iconfont: 'mdi'
-})
+Vue.use(Vuetify)
 Vue.use(Vuelidate)
 
+addDecorator(() => ({
+  vuetify: new Vuetify({ iconfont: 'mdi' }),
```

Comment:
This little line here was required to "configure" Vuetify to work correctly. Before this I was getting an error, "Cannot read property 'dark' of undefined at VueComponent.appIsDark".